zebra-state/src/service/finalized_state/zebra_db/shielded.rs

```diff
@@ -224,7 +260,9 @@ impl ZebraDb {
         NoteCommitmentTrees {
             sprout: self.sprout_tree(),
             sapling: self.sapling_tree(),
+            sapling_subtree: None,
             orchard: self.orchard_tree(),
+            orchard_subtree: None,
```

Comment:
Looks like you might need `*_subtree_for_tip()` methods here.

One way to get them is `zs_last_key_value()`, feel free to grab the `impl FromDisk for NoteCommitmentSubtreeIndex` from PR #7408.

Reply:
This is now ticket #7444